lib/bazel_worker.dart

```diff
@@ -2,10 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-export 'src/async_worker_loop.dart';
+export 'src/worker/async_worker_loop.dart';
```

Comment:
is it feasible to first submit a commit that moves these files so the diff is less noisy?